### PR TITLE
fix: Loading spinner is not centered in the authentication providers 

### LINKED ISSUE #1368: Loading spinner is not centered in the authentication providers container
The spinner should be centred. This bug was introduced with the recent update.

image

### PR DESCRIPTION
**Pull request by talwinder50**
Screen Shot 2021-12-22 at 10 45 45 AM
Screen Shot 2021-12-22 at 10 45 57 AM

closes #1368

Signed-off-by: talwinder50 <talwinderkaur50@gmail.com>

### CODE REVIEW

File: cmd/wallet-web/src/pages/Signin.vue [hunk 1]
```diff
@@ -49,14 +49,27 @@
...
+          class="
+            grid grid-cols-1
+            gap-5
+            w-full
+            sm:px-32
+            h-64
+            mb-12
+            items-center
+            content-center
+            justify-items-center
```
Comment (birtony) [inline cmd/wallet-web/src/pages/Signin.vue:62]: This is not a tailwind class

File: cmd/wallet-web/src/pages/Signup.vue [hunk 2]
```diff
@@ -79,14 +79,17 @@
                   {{ t('Signup.heading') }}
                 </h1>
               </div>
-              <div class="grid grid-cols-1 gap-5 w-full h-64 mb-8 content-center">
+              <div
+                class="grid grid-cols-1 gap-5 w-full h-64 mb-8 content-center justify-items-center"
```
Comment (birtony) [inline cmd/wallet-web/src/pages/Signup.vue:83]: `justify-items-center` is not a tailwind class

File: cmd/wallet-web/src/pages/Signin.vue (hunk 1 above)
Comment (birtony) [reply cmd/wallet-web/src/pages/Signin.vue:62]: I assume what you wanted is `items-center`, which is already there two lines above. However, it only work inside flex display, this container has grid

**Review by birtony — CHANGES_REQUESTED** [review]

File: cmd/wallet-web/src/pages/Signup.vue (hunk 2 above)
Comment (talwinder50) [reply cmd/wallet-web/src/pages/Signup.vue:83]: https://tailwindcss.com/docs/justify-items it is tailwind class, may be version is the issue

File: cmd/wallet-web/src/pages/Signin.vue (hunk 1 above)
Comment (talwinder50) [reply cmd/wallet-web/src/pages/Signin.vue:62]: yes i need grid and using flex here gives Classnames grid, flex are conflicting errors

**Review by rolsonquadras — APPROVED** [review]

**Review by birtony — APPROVED** [review]

**Pull request merged**